docs/subnets/add-subnet.md

```diff
@@ -16,8 +16,15 @@ required_permissions:
 1. On Devopness, navigate to a project then select an environment
 1. Find the `Networks` card
 1. Click `View` in the `Networks` card, to see a list of existing `Networks`
-1. In the list of networks, find the network you want to manage and click `DETAILS`
+1. In the list of networks find the network where you want to add a subnet and click the `NAME` of the network
 1. On the upper-right corner of network details view, click `SUBNETS`
 1. On the upper-right corner of subnets list, click `ADD SUBNET`
-1. Follow the prompts then click `CONFIRM`
+1. Click `NEXT`
+1. Provide a name to the Network being added. Example: "Automate Net Subnet"
```

Comment:
Perhaps give simpler examples for the subnet name?

```suggestion
1. Provide a name to the Subnet being added. Example: "Public Subnet A", "Private Subnet B"
```

Reply:
Great, fixed!